src/domain/usecase/user/login.ts

```diff
@@ -51,6 +51,11 @@ class LoginUseCase implements UseCase<LoginInput, Result<LoginOutput, Error>> {
         }
 
         const { password: userPwd, salt: userSalt } = user;
+
+        if (!userPwd || !userSalt) {
+            return err(new Error('소셜 로그인 유저'));
+        }
```

Comment:
GoolgleAuth에서 이미 다 알아서 가입 혹은 로그인을 시키는데, 이 부분이 필요한 이유가 어떤거지?

Reply:
null check 를 안하면 trpescript 에서 경고가 떠서..

Reply:
```javascript
const { password: userPwd, salt: userSalt } = user;
```
이 부분을 밑에 처럼 바꾸면 저렇게 체크 안 해도 되긴 할 텐데... 필요할 수도 있겠군...
```suggestion
        const userPwd = user.password!;
        const userSalt = user.salt!;
```